NakedJavaScript/mensa-projekt
# [M05]Details vom Essen anzeigen

# What has changed:
* After adding a meal to a day in the index page, the details will be shown (of the meal)
* When the fields for "allergens" or "etc are empty when submitting, a default value will be added instead.
# How to test:
* Try to add a meal to a day on the index page, you should now see the details of it
* Create a meal without empty fields for allergens or etc, after the creation you should see a default value for this field.
